### PR TITLE
adjust btn in instruments/show.html.erb and add booking create action

### PR DESCRIPTION
**Pull request by phiele**
Adjust btn in instruments/show.html.erb and add booking create action. 

The redirection after a successful booking still needs to be implemented, but this depends on the user authentification, so we should implement next.

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -5,5 +5,21 @@ def new
   end
 
   def create
+    @instrument = Instrument.find(params[:instrument_id])
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
```
Comment (rudygasson) [inline app/controllers/bookings_controller.rb:10]: You could just add:
`before_action :authenticate_user!` in the first line of the controller, then all controller actions here need an authenticated user

**Review by rudygasson — COMMENTED** [review]
OK, so Soma added the authentication in another PR, so that should work after merging. Add the button style change is now doubled, but... well.

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -5,5 +5,21 @@ def new
   end
 
   def create
+    @instrument = Instrument.find(params[:instrument_id])
```
Comment (rudygasson) [inline app/controllers/bookings_controller.rb:8]: Is the param key actually `instrument_id` here? Just wondering...

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (phiele) [reply app/controllers/bookings_controller.rb:10]: Exactly, we added the authentication the other pull request

**Review by somazsoter — APPROVED** [review]
Looks good!

**Pull request merged**